signac/contrib/project.py

```diff
@@ -1135,11 +1135,12 @@ def groupby(self, key=None, default=None):
             present. The value must be sortable and is only used if not None
             (Default value = None).
 
-        Returns
-        -------
-        Tuple[str, Iterable[Job]]
-            A pair whose first element is the grouped key and whose second element
-            is an iterable of `Job` instances matching the key.
+        Yields
```

Comment:
Should we use `yield from` instead of `return` for the `itertools.groupby` object? Is it clear that this acts as a generator?

The `itertools.groupby` object has no useful methods beyond ones like `__next__` and `__iter__` that define it as an iterable.

Reply:
I would be fine with that switch rather than documenting that we returning an `itertools.groupby` object.

Reply:
Changed.